index.js

```diff
@@ -169,7 +169,7 @@ jasmine.Expectation.prototype.wrapCompare = function(name, matcherFactory) {
 
     if (!webdriver.promise.isPromise(expectation.actual) &&
         !webdriver.promise.isPromise(expected)) {
-      originalWrapCompare(name, matcherFactory).apply(this, arguments);
+      compare(expectation.actual, expected);
```

Comment:
I believe we can remove `originalWrapCompare` above now, since it's now  unused.